tools/autodeployment/server.js

```diff
@@ -26,3 +26,11 @@ handler.on('push', (event) => {
     shell.exec('./deploy_stage.sh');
   }
 });
+
+handler.on('release', (event) => {
```

Comment:
Will the `push` handler above not also get triggered all the time?  We'd need a way to ignore that.  We could use an environment variable, and use an `if` to look at `process.env.DEPLOYMENT === 'staging'` vs. `process.env.DEPLOYMENT === 'production'`, and only register the `handler` event for the appropriate situation.

Reply:
I just thought of this properly, you're right. If the if statement above gets executed, it'll wipe the whole telescope folder.